src/CollectdWinService/ReadWindowsAttributesPlugin.cs

```diff
@@ -74,13 +74,8 @@ public IList<CollectableValue> Read()
             var collectedValueList = new List<CollectableValue>();
 
             collectedValueList.AddRange(GetCommonAttributes());
+            collectedValueList.AddRange(GetEC2Metadata());
 
-            if (_readEC2InstanceMetadata)
```

Comment:
Looks like this is actually a config, so can you just remove line 82 and leave the if statement?

https://github.com/Netuitive/netuitive-windows-agent/blob/f18aaefb163ddb86733f713b7354781248e2cc7b/src/CollectdWinService/ReadWindowsAttributesPlugin.cs#L44